packages/components/dropdowns/dropdown-menu/src/trigger/dropdown-menu-trigger.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode, forwardRef } from 'react';
+
+export type TDropdownTriggerProps = {
+  onClick: () => void;
+  children: ReactNode;
+};
+const DropDownTrigger = forwardRef<HTMLDivElement, TDropdownTriggerProps>(
```

Comment:
nit: 
```suggestion
const DropdownTrigger = forwardRef<HTMLDivElement, TDropdownTriggerProps>(
```
(for consistency)